tests/LinkWrapperOverrideTest.php

```diff
@@ -6,7 +6,17 @@
 
 class LinkWrapperOverrideTest extends DeltaTestCase
 {
-    public $json = '[{"attributes":{"link":"https://luya.io"},"insert":"luya.io"},{"insert":" test\n\nFooter\n"}]';
+    public $json = [
+        [
+            'attributes' => [
+                'link' => 'https://luya.io',
+            ],
+            'insert' => 'luya.io',
+        ],
+        [
+            'insert' => ' test'.PHP_EOL.PHP_EOL.'Footer'.PHP_EOL,
```

Comment:
I also converted from double to single quotes since that is what we use in php. This also requires us to use PHP_EOL.

I don't mind. But we could of course also use double quotes for these cases.